Source/WebKit/UIProcess/mac/WebViewImpl.h

```diff
@@ -907,7 +910,7 @@ class WebViewImpl final : public CanMakeWeakPtr<WebViewImpl>, public CanMakeChec
 #endif
 
 #if ENABLE(IMAGE_ANALYSIS)
-    CocoaImageAnalyzer *ensureImageAnalyzer();
+    RetainPtr<CocoaImageAnalyzer> ensureProtectedImageAnalyzer();
```

Comment:
This goes against our usual style. This should keep returning a raw pointer for call sites that do not need a smart pointer. And you can add a new `ensureProtectedImageAnalyzer()` for call sites that do need a smart pointer.

Reply:
It seems that it's never used in a place where a smart pointer wouldn't be needed. But I am going to leave the original raw getter and remove the protected one for now, I think that after marking the member as `const` this warning will go away without needing a protected getter.